calcom/cal.com
# fix: Creating a default availability for org owner [CAL-2142]

## What does this PR do?

When a user creates an organization, they don't go through the profile onboarding, preventing them to have a default availability created for them.

This PR introduces the creation of a default availability when the owner account is created. Empty state was already there, just not accessible through UI.

## Type of change


## How should this be tested?

1. Create an organization
2. Go to the Availability page
3. See a default availability be created

> **Note**
> As you will not be able to delete the only availability to see the empty state, check it deleting availability from DB

## Mandatory Tasks
